### PR TITLE
Added Servlet to get drug info.

### PR DESCRIPTION
**Pull request by GMarsh0218**
ListDrugInfoServlet returns all the information about a Drug when passed with the name parameter.

### CODE REVIEW

File: src/main/java/com/google/sps/servlets/ListDrugInfoServlet.java [hunk 1]
```diff
@@ -35,38 +32,36 @@
...
+public class ListDrugInfoServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     
     Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
-    String drugname = request.getParameter()
-    Query<Entity> query =
-        Query.newEntityQueryBuilder().setKind("Drug").setOrderBy(OrderBy.asc("Name")).build();
+    String gqlQuery = "SELECT * FROM Drug WHERE Name = " + request.getParameter("Name");
```
Comment (EdithBenvenuto) [inline src/main/java/com/google/sps/servlets/ListDrugInfoServlet.java]: I think the parameter is sending is the id

Comment (GMarsh0218) [reply src/main/java/com/google/sps/servlets/ListDrugInfoServlet.java]: oh okay, fixing it now

Comment (GMarsh0218) [reply src/main/java/com/google/sps/servlets/ListDrugInfoServlet.java]: Sorry, thats outdated check Files Changed for the current change.

Comment (EdithBenvenuto) [reply src/main/java/com/google/sps/servlets/ListDrugInfoServlet.java]: okay

**Review by EdithBenvenuto — APPROVED** [review]

File: src/main/java/com/google/sps/servlets/ListDrugInfoServlet.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+public class ListDrugInfoServlet extends HttpServlet {
+
+	@Override
+	public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
+    String Name = request.getParameter("Name");
+    Query<Entity> query = Query.newEntityQueryBuilder()
+            .setKind("Drug")
+            .setFilter(StructuredQuery.PropertyFilter.eq("Name", Name))
```
Comment (mcollins42) [inline src/main/java/com/google/sps/servlets/ListDrugInfoServlet.java:41]: I'm still seeing a query by name, not a lookup by key ID which is simpler and more efficient.

**Review by mcollins42 — COMMENTED** [review]
I don't see the update to ListDrugInfoServlet to lookup by ID.

**Pull request merged**